src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer work!");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer stop!");
+    }
+}
```

Comment:
```suggestion
public class Bulldozer extends Machine {
    @Override
    public void doWork() {
        System.out.println("Bulldozer work!");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer stop!");
    }
}
```
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)
**Please don’t add redundant empty lines to your code.**

Same for other classes.

Reply:
Empty lines are used: 1. between methods, 2. between local variables and first command, 3. before comments, 4. before the class declaration, 5. at the end of the class (to avoid mistake /n). Am I right?
I have completed the fix about redundant empty lines.